packages/dev/docs/pages/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Allow: /
```

Comment:
hmm.... where do our assets go? do we want to exclude those?

Reply:
should be fine i think? I dont think js and such get indexed